<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui2</class>
 <widget class="QMainWindow" name="gui2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Acerta se puderes</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_led1">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #2da44e;
	color: #f5f5f5;
	border: 1px solid transparent;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: #1b8237;
}</string>
    </property>
    <property name="text">
     <string>Led 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_led2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #df0024;
	color: #f5f5f5;
	border: 1px solid transparent;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: #c1001f;
}</string>
    </property>
    <property name="text">
     <string>Led 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_led3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>210</y>
      <width>200</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	background-color: #075ede;
	color: #f5f5f5;
	border: 1px solid transparent;
	border-radius: 6px;
}
QPushButton:hover {
	background-color: #094aaa;
}</string>
    </property>
    <property name="text">
     <string>Led 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_advinhe">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>Advinhe! Qual dos leds estará aceso?</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>471</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nome do jogador:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_pontuacao">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Open Sans</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(245, 245, 245);</string>
    </property>
    <property name="text">
     <string>Pontuação:</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
